app/src/main/res/drawable/rectangle_background.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.rectangle_background` appears to be unused
 Unused resources make applications larger and slow down builds.